server/app/playbook_run.go

```diff
@@ -475,9 +475,9 @@ type PlaybookRunService interface {
 	// NukeDB removes all playbook run related data.
 	NukeDB() error
 
-	// SetReminder sets a reminder. After timeInMinutes in the future, the owner will be
-	// reminded to update the playbook run's status.
-	SetReminder(playbookRunID string, timeInMinutes time.Duration) error
+	// SetReminder sets a reminder. After fromNow (time.Now().Add(fromNow)) in the future,
```

Comment:
Super nit:
```suggestion
	// SetReminder sets a reminder. After `time.Now().Add(fromNow)` in the future,
```

I first read this as `fromNow` being a function and `time.Now().Add(fromNow)` being an argument. Probably because it's late and I'm tired, but I guess it doesn't hurt to mention.

Reply:
Hah, I see it, fixed. :)